mscore/workspace.cpp

```diff
@@ -823,9 +823,9 @@ void Workspace::read(XmlReader& e)
                   QMenuBar* mb = mscore->menuBar();
                   const QObjectList menus(mb->children()); // need a copy
                   for (QObject* m : menus) {
-                        if (qobject_cast<QMenu*>(m)) {
-                              m->setParent(nullptr);
-                              m->deleteLater();
+                        if (QMenu* menu = qobject_cast<QMenu*>(m)) {
```

Comment:
What exactly the reason of crash?

And why QMenuBar::clear() isn't enough? Did you tested it without 827 and 828?

Please, move local variable from the if condition.

Reply:
I've just turned the patch provided in  https://musescore.org/en/node/299654 into a PR.
Changes to it should better discussed there I guess?

Reply:
Yeah, sorry, didn't noticed that he attached the patch.